content/neuvector-prime/azure/contents.lr

```diff
@@ -0,0 +1,476 @@
+           "controller.secret.data.userinitcfg\\.yaml.users[0].Role"
+
+    Changing this value does not update the NeuVector Admin Fullname, Password,
+    or Role. This will cause unnecessary cluster restarts.
+
+    ---
+
+
+
+
+
+
+
+   !["Keyvaluepair config"](./keyvaluepairconfig.png)
+
+## NeuVector Usage Record and Billing
+
+This command will print out CSP Adapter Usage Record:
+
+```
+kubectl get CspAdapterUsageRecord  neuvector-usage -n neuvector -oyaml
+```
+
+it should list similar output
+
+```
+kubectl get CspAdapterUsageRecord  neuvector-usage -n neuvector -oyaml
+apiVersion: susecloud.net/v1
+base_product: cpe:/o:suse:neuvector:5.2.0
+kind: CspAdapterUsageRecord
+managed_node_count: 1
+metadata:
+  creationTimestamp: "2023-09-12T16:29:56Z"
+  generation: 3
+  name: neuvector-usage
+  resourceVersion: "2057534"
+  uid: e12a5890-b97e-4fef-9373-242ccce0a14c
+reporting_time: "2023-09-12T16:32:27.012177+00:00"
+```
+
+Billing will be available in Azure Portal billing
+
+Home > Cost Management <subscription> | Cost analysis
+
+
+## Uninstalling NeuVector and Scanner Updater
+
+Before Uninstalling NeuVector, uninstall the scanner updater.
+Execute the following command in a Cloud Shell or Node which has access to kubectl.
+
+```
+kubectl delete -f scanner.yaml
+```
+it should list similar output
+```
+deployment.apps "neuvector-scanner-pod" deleted
+cronjob.batch "neuvector-updater-pod" deleted
+```
+
+Check the scanner pods are running:
+```
+kubectl get pods -n neuvector
+```
+it should not list neuvector-scanner-pod in the output.
+
+```
+NAME                                        READY   STATUS    RESTARTS   AGE
+neuvector-controller-pod-754c55f668-5vnxv   1/1     Running   0          3h7m
+neuvector-controller-pod-754c55f668-658vb   1/1     Running   0          3h8m
+neuvector-controller-pod-754c55f668-bfk2c   1/1     Running   0          3h10m
+neuvector-csp-pod-cb675b47c-wnm8g           1/1     Running   0          3h15m
+neuvector-enforcer-pod-2d9qd                1/1     Running   0          3h15m
+neuvector-enforcer-pod-5lgw4                1/1     Running   0          3h15m
+neuvector-enforcer-pod-8qxdb                1/1     Running   0          3h15m
+neuvector-enforcer-pod-9gngc                1/1     Running   0          3h15m
+neuvector-enforcer-pod-9jnt7                1/1     Running   0          3h15m
+neuvector-manager-pod-59677c8ccb-4g9hj      1/1     Running   0          3h10m
+```
+
+To navigate to and uninstall NeuVector in the Azure Portal
+
+
+1. Click Home > Kubernetes Services
+2. Click the Cluster name
+3. Click Settings > Extension + application
+4. Select NeuVector Extension to Uninstall
+
+
+
+!["Uninstall extension"](./uninstall.png)
+
+
+
+## Troubleshooting
+
+This section contains information to help you troubleshoot issues when install NeuVector and configure Billing-adapter
+
+After successful deployment, it should list similar pod and chart output
```

Comment:
After successful deployment, the deployments list should include pod and chart output similar to the following example: